index.js

```diff
@@ -63,76 +60,42 @@ var winner
      if (currentplayer === player1) {
           var x= prompt("Player1 Introduce the coordenates x of your opponent ship", "0");
           var y = prompt("Player1 Introduce the coordenates y of your opponent ship", "0"); 
```

Comment:
When indenting, 2 or 4 spaces is recommended (3 is weird). 
We should use the same amount of spaces when indenting throughout all your code (if we start with 2 spaces, we indent always with 2 spaces, same principle with 4 spaces).